packages/core/lib/middleware/incoming_request_data.d.ts

```diff
@@ -0,0 +1,11 @@
+import * as http from 'http';
+
+declare class IncomingRequestData {
+  request: { [key: string]: any };
+
+  constructor(req: http.IncomingMessage);
```

Comment:
why is the `req` only typed as `http.IncomingMessage` and not `http.IncomingMessage | https.IncomingMessage`